packages/global/components/blocks/content-load-more.marko

```diff
@@ -39,10 +41,10 @@ $ const {
   <marko-web-load-more
     header=`More in ${sectionName}`
     component-name="content-load-more-flow"
-    component-input={ aliases, nativeX: input.nativeX }
+    component-input={ primarySectionAlias, aliases, nativeX: input.nativeX }
```

Comment:
@Shinsina  disableAds should also be passed to all three content-load-more-flow calls.